src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -93,7 +93,7 @@ public Task StartAsync(Uri url, IDuplexPipe application, TransferMode requestedT
                     {
                         var result = await pipelineReader.ReadAsync();
                         var input = result.Buffer;
-                        if (result.IsCancelled || (input.IsEmpty && result.IsCompleted))
+                        if (result.IsCanceled || (input.IsEmpty && result.IsCompleted))
```

Comment:
There was talk in the demo meeting about a new API for something like HasData (I don't remember what was actually said). Coming later?

Reply:
Definitely not in this iteration.